src/tests/auth/login.test.js

```diff
@@ -0,0 +1,190 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import models from '../../models';
+import app from '../../app';
+import {
+  activeUser,
+  notActive,
+  wrongToken,
+  tokenWithWrongUser,
+  wrongUser
+} from '../fixtures/users.fixture';
+
+chai.use(chaiHttp);
+const router = () => chai.request(app);
+
+const { Users } = models;
+
+let user = null;
+let token = null;
+const userPassword = 'Pass1234';
+
+
+describe('Tests for user login', () => {
+  before(async () => {
+    await Users.create(activeUser);
+    await Users.create(notActive);
+    user = {
+      email: activeUser.email,
+      password: userPassword,
+    };
+  });
+  it('should login a user', (done) => {
+    router()
+      .post('/api/auth/login')
+      .send(user)
+      .end((err, res) => {
+        token = res.body.data;
+        expect(res).to.have.status(200);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('data');
```

Comment:
Check that the data returned is the same as the request payload

Reply:
I am returning token, so it would be necessary for me to verify that token. Should I do that?

Reply:
Is the token the only thing you are returning? If yes, you don't have to verify the token.

Reply:
Yeah it is the only thing I am returning